src/components/molecules/card/card.scss

```
@@ -0,0 +1,55 @@
+@use "../../variables" as *;
+.card {
+  border: 0px;
+
+  .card-img {
+    border-radius: 0;
+    position: relative;
+
+    .triangle {
+      position: absolute;
+      border-right: 150px solid transparent;
+      border-top: 150px solid white;
+      z-index: 1;
+    }
+
+    .title {
+      position: absolute;
+      font-family: $fontfamily-gothic;
+      font-size: $fontsize-l;
+      font-weight: 700;
+      color: $orange_bistre;
+      z-index: 2;
+      margin-left: 6px;
+      margin-bottom: 0px;
+    }
+
+    .img {
+      overflow: hidden;
+    }
+    img {
+      width: 100%;
+      transition: 0.5s;
+      @media screen and (min-width: $breakpoint-md) {
+        &:hover {
+          transform: scale(1.2);
+        }
+      }

```

```suggestion
    .img {
      overflow: hidden;
          img {
      width: 100%;
      transition: 0.5s;
      @media screen and (min-width: $breakpoint-md) {
        &:hover {
          transform: scale(1.2);
        }
      }
    }
```
HTMLの階層構造はSCSSにも反映させるようにお願いします。
今回であれは上のようになると思います。
（`.img` というクラス名を変更してほしいというのは置いておいて）